mixer/pkg/lang/cel/builder_test.go

```diff
@@ -626,9 +626,34 @@ var (
 			// note that type lookup is a runtime operation, so the attribute lookup is necessary
 			referenced: []string{"-context.reporter.kind"},
 		},
+		//Taken from cel-go strings readme: https://github.com/google/cel-go/tree/master/ext
+		{
+			text:       `source.name.replace('he', 'we')`,
+			bag:        map[string]interface{}{"source.name": "hello hello"},
+			result:     "wello wello",
+			referenced: []string{"source.name"},
+		},
+		{
+			text:       `source.name.replace('he', 'we', -1)`,
+			bag:        map[string]interface{}{"source.name": "hello hello"},
+			result:     "wello wello",
+			referenced: []string{"source.name"},
+		},
+		{
+			text:       `source.name.replace('he', 'we', 1)`,
+			bag:        map[string]interface{}{"source.name": "hello hello"},
+			result:     "wello hello",
+			referenced: []string{"source.name"},
+		},
+		{
+			text:       `source.name.replace('he', 'we', 0)`,
+			bag:        map[string]interface{}{"source.name": "hello hello"},
+			result:     "hello hello",
+			referenced: []string{"source.name"},
+		},
 	}
 	attrs = map[string]*v1beta1.AttributeManifest_AttributeInfo{
-		"as":                        {ValueType: v1beta1.STRING},
+		"abc":                       {ValueType: v1beta1.STRING},
```

Comment:
I'm not sure you really want to change the name here. Seems like maybe there was a replaceAll that ran astray.

Reply:
This one does need to be changed - some of the tests in TestCELExpressions were looking for the attribute `as` - and fail with `undeclared reference to 'abc'` if this isn't changed.

Reply:
I'm fine with the rename of "as" to "abc". I think it's a good idea not to use potential keywords as variable names.